[AIRFLOW-173] - Initial implementation of FileSensor

trixpan opened the pull request:
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-173

Implement a simple FileSensor to detect the creation of files and folder
in a given filesystem (a la HDFSSensor)

bolkedebruin commented on airflow/hooks/fs_hook.py:
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
This file needs to go to contrib. As does its operator

trixpan committed:
AIRFLOW-173 - Initial implementation of FileSensor
Implement a simple FileSensor to detect the creation of files and folder
in a given filesystem (a la HDFSSensor)

Anmolk7 reviewed (commented):
the FileSensor doesnt sense file. The task terminates as success even though there is no filepath given or a non existing filepath is given to the extra field in the connections
